### PR TITLE
Support IntelliJ 2020.2 EAP

### LINKED ISSUE #35: Support IntelliJ 2020.2-EAP

### PR DESCRIPTION
**Pull request by pgilad**
Closes #35 

- Update plugin to support 2020.2-EAP
- Remove `.idea` files from git
- Update dependencies and used plugins
- Update Python plugin
- Update to latest Gradle (6.5)

A sample release can be found here: https://github.com/pgilad/requirements/releases/tag/2020.2-202

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,32 +41,13 @@ hs_err_pid*
...
-.idea/**/sqlDataSources.xml
-.idea/**/dynamic.xml
-.idea/**/uiDesigner.xml
-.idea/**/dbnavigator.xml
-
-# Gradle
-.idea/**/gradle.xml
-.idea/**/libraries
+.idea/
+*.iml
```
Comment (meanmail) [inline .gitignore:45]: It is controversial to delete Idea files. After all, this project is unlikely someone will open outside Idea

File: build.gradle.kts [hunk 2]
```diff
@@ -44,16 +43,14 @@ intellij {
     version = if (project.properties["eap"].toString() == "true") {
         "LATEST-EAP-SNAPSHOT"
     } else {
-        project.properties["IdeVersion"].toString()
+        project.properties["ideaVersion"].toString()
```
[the remark this answers is not in the document]
Comment (pgilad) [reply build.gradle.kts:46]: Will revert

File: .gitignore (hunk 1 above)
Comment (pgilad) [reply .gitignore:45]: If you leave idea files, you are forcing me to configure IntelliJ the way you do as well, but that's not very friendly. I can restore the files, but usually it's not a good practice for common library to git add IDE files, from what I know it's a practice more fitting to internal team projects

**Comment by meanmail** [comment]
The pull request is outdated

**Pull request closed without merging**